lib/rules/focus-order-semantics.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "focus-order-semantics",
+  "selector": "div, h1, h2, h3, h4, h5, h6, [role=heading], p, span",
```

Comment:
I don't quite understand what `[role=heading]` is doing in here. What error are you expecting to catch?

Reply:
e.g. em[role=heading][tabindex=1]